SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -544,6 +544,25 @@ void USpatialNetDriver::CreateServerSpatialOSNetConnection()
 	GetWorld()->SpatialProcessServerTravelDelegate.BindStatic(SpatialProcessServerTravel);
 }
 
+void USpatialNetDriver::CleanUpServerConnectionForPC(APlayerController* PC)
+{
+	// Can't do Cast<USpatialNetConnection>(PC->Player) as Player is null for some reason.
```

Comment:
Turned out the reason was that we explicitly set it to null in `USpatialReceiver::RemoveActor` ([code](https://github.com/spatialos/UnrealGDK/blob/22c40a36ed37472d09d453c3e55bd6047222ea20/SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp#L1106-L1110), might want to update the comment.

Reply:
As discussed with you and tested. Its null even without this nulling in RemoveActor so leaving as is.